Callout that impersonation needs (ClusterRole)Binding (#34082)
kubernetes/website: content/en/docs/reference/access-authn-authz/authentication.md

  resourceNames: ["06f6ce97-e2c5-4ab8-7ba5-7654dd08d52b"]
```

{{< note >}}

> Note that impersonation is not namespace scoped - i.e. it requires a `ClusterRole` and `ClusterRoleBinding`, and does not function with `Role` and `RoleBinding`.

Reviewer: ```suggestion
Note that impersonation is not namespace scoped. It requires a `ClusterRole` and `ClusterRoleBinding`,
and does not function with `Role` and `RoleBinding`.
```

  resourceNames: ["06f6ce97-e2c5-4ab8-7ba5-7654dd08d52b"]
```

{{< note >}}

> Note that impersonation is not namespace scoped. It requires a `ClusterRole` and `ClusterRoleBinding`,

Reviewer 2: nit: watch out for starting a note callout with “Note that”
```suggestion
Impersonating a user allows you to peform any action as that user; for that reason, impersonation
is not namespace scoped. If you want to allow impersonation using Kubernetes RBAC, 
this requires using a `ClusterRole` and a `ClusterRoleBinding`,
```

  resourceNames: ["06f6ce97-e2c5-4ab8-7ba5-7654dd08d52b"]
```

{{< note >}}
Impersonating a user (or group) allows you to perform any action as if you were that user (or group);
for that reason, impersonation is not namespace scoped.
If you want to allow impersonation using Kubernetes RBAC, 

> this requires using a `ClusterRole` and a `ClusterRoleBinding`,

Reviewer 3: ```suggestion
use a `ClusterRole` and a `ClusterRoleBinding`,
```
